docs/source/getting-started/path-variables.rst

```diff
@@ -0,0 +1,113 @@
+Setting JAVA_HOME and PATH variables
+##################
+
+Windows
+***************************
+We'll explain here how to set up ``JAVA_HOME`` and ``PATH`` variables on Windows.
+
+Setting the JAVA_HOME variable
+=============
+
+1. Open ``Search`` and type "advanced system settings" and hit ``Enter``.
+2. Find the ``Advanced`` tab and click on the ``Environment Variables``.
+3. In the ``System variables`` section click on ``New`` if you want to set ``JAVA_HOME`` system wide. To set ``JAVA_HOME`` for a single user click on ``New`` under ``User variables``.
+4. In the opened modal window for ``Variable name`` type ``JAVA_HOME`` and for the ``Variable value`` set the path to the JDK installation. Click OK once you're done.
+5. Click OK and click Apply to apply the changes.
+
+Setting the PATH variable
+=============
+If you have downloaded and installed the DAML SDK using our `Windows installer <https://github.com/digital-asset/daml/releases/latest>`_ your ``PATH`` variable is already set up.
+
+Mac OS
+***************************
+We'll explain here how to set up ``JAVA_HOME`` and ``PATH`` variables on Mac OS with ``bash`` console.
+If yu are using ``zsh`` all of the instructions are quite similar, except that you will be doing all of the changes
+in the ``.zshrc`` file.
+
+Setting the JAVA_HOME variable
+=============
+Open the ``~/.bash_profile`` in any text editor. The file should be located in your ``Home`` folder.
+In ``Finder`` click on Go --> Home and then click ``Cmd+SHIFT+.`` to see the hidden files. Locate the ``~/.bash_profile``
+and double click on it to open it. Add the folloiwing line as a new line at the end of the file::
+
+        export JAVA_HOME=$(/usr/libexec/java_home)
+
+Save the file before closing.
+
+Next open the terminal and run the source command to apply the changes::
```

Comment:
I’ll leave it up to you but I would simply suggest to reboot or at least log in again. Figuring out when a change applies to which terminal is just annoying and a source of a lot of issues.

Reply:
Agreed. People who know how the `PATH` variable works won't need to do this, but they're not reading this article.

Reply:
Thnx @cocreature and @SamirTalwar-DA. Changed the instructions so that users should reboot.